### ISSUE TITLE
“The file nuget.config was not found” dialog 

### ISSUE THREAD

**Issue by petermorlion** [NuGet/Home#1789]
I often get this warning when opening a Solution in Visual Studio:

![Nuget.config not found dialog](http://i.stack.imgur.com/yzqOd.png)

I get this several times, but when I just click through the dialogs, everything opens fine and builds fine, even on other developer's PC's.

Our nuget.config file actually does exist, but is not in the solution's directory. It is actually up some directories, because we share it across solutions.

I believe this has to do with the Package Manager Console. If it is opened when I open the solution, I get the error. If it isn't open, I don't.

However, I haven't been able to reproduce this with a new solution. So I'm guessing it must be something local, but I have no idea where to look. We're not working with NuGet Package Restore, our nuget.config file looks normal and so does our repositories.config.

This is our nuget.config file:

```
<?xml version="1.0" encoding="utf-8"?>
<configuration>
    <config>
        <add key="repositorypath" value="./References" />
    </config>
</configuration>
```

However, when I add an empty nuget.config file next to the solution file (.sln file), I no longer get the error.

I'm at a loss here. I can't find this error in the source, nor can I find the CheckForNugetConfig package anywhere (NuGet, Google, source).

Any ideas?

**Comment by yishaigalatzer** [comment]
What version of NuGet and Visual Studio are you using?

**Comment by petermorlion** [comment]
Hi yishaigalatzer, I'm using:
- Visual Studio Premium 2013, Version 12.0.31101.00 Update 4
- NuGet Package Manager 2.8.50926.663

**Comment by yishaigalatzer** [comment]
I have a few suggestions
1. If possible move to 2015 NuGet 3.X, where we actively develop and bug fix every item. This seems like a lower priority thing, we can probably find a workaround for.
2. Try creating an empty nuget.config at the solution level (each solution), it will perhaps make nuget just keep working.
3. Sharing the repository between multiple solution is fraught with trouble, and not officially supported. You can instead just use the default repository path. No I don't have a good example, I just know there are no good solid test cases to cover the scenario

If that doesn't work, we can try out more ideas.

**Comment by yishaigalatzer** [comment]
Hey @petermorlion  did you have a chance to look at the suggestions?

**Comment by petermorlion** [comment]
Ah, sort of lost track of this, sorry.
This is on a company PC where I don't get to choose when to update to which version, unfortunately. But I have passed your comments on to those responsible. With a small group of devs, we've been trying to get them to use NuGet as it's intended (with a packages folder next to the solution). Your comments should help our case :)
I'll probably be re-organizing my folders to use the industry/NuGet-standard way.

But I thougt NuGet did support this, as it searches up the directory tree until it finds a nuget.config file, no?

I have tried adding an empty nuget.config file next to the solution. It didn't give the error, but it said the packages needed to be restored. So it seems, it doesn't search up the directory tree anymore.
